internal/infrastructure/db/repositories/device_definition_repo.go

```diff
@@ -123,6 +124,24 @@ func (r *deviceDefinitionRepository) GetAll(ctx context.Context) ([]*models.Devi
 	return dd, err
 }
 
+func (r *deviceDefinitionRepository) GetDeviceMMY(ctx context.Context) ([]*models.DeviceDefinition, error) {
+
+	dd, err := models.DeviceDefinitions(
```

Comment:
to make this query more efficient, b/c it is pulling lots of data, let's only select the columns we care about. To do that, [here is an example](https://github.com/volatiletech/sqlboiler#select)

Reply:
I'm having trouble testing that this query returns what I want it to-- can you take a look at it?